app/models/application_record.rb

```diff
@@ -37,15 +32,6 @@ def self.sanitize_for_search(term, **cols)
     ActiveRecord::Base.send(:sanitize_sql_array, ["MATCH (#{cols}) AGAINST (? IN BOOLEAN MODE)", term])
   end
 
-  # From http://stackoverflow.com/questions/6591722/how-to-generate-fixtures-based-on-my-development-database
-  def dump_fixture
-    fixture_file = "#{Rails.root}/test/fixtures/#{self.class.table_name}.yml"
-    File.open(fixture_file, 'a') do |f|
-      f.puts({ "#{self.class.table_name.singularize}_#{id}" => attributes }
-        .to_yaml.sub!(/---\s?/, "\n"))
-    end
-  end
-
```

Comment:
This method is no longer being used.

Reply:
Ah - this one does need to stay. It's not used in normal operation, no, but it's there to provide easy seeding for development.